stl/inc/xstring

```diff
@@ -4600,6 +4634,19 @@ template <class _Elem, class _Traits, class _Alloc>
 _NODISCARD bool operator>=(const basic_string<_Elem, _Traits, _Alloc>& _Left, _In_z_ const _Elem* const _Right) {
     return !(_Left < _Right);
 }
+#else // !_HAS_CXX20

```

Comment:
We conventionally annotate `#else` with descriptors for both the section above and below. Here, that would look like:
```suggestion
#else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
```
indicating the code above is active when `!_HAS_CXX20` is true, and the code below is active when `_HAS_CXX20` is true.